sendfile/views.py

```diff
@@ -0,0 +1,20 @@
+from django.views.generic.detail import BaseDetailView
+from sendfile import sendfile
+
+class BaseXSendFileView(BaseDetailView):
```

Comment:
I don't think this is a base view, the only things I need to provide can be passed in via kwargs to `as_view`.

Also, a docstring explaining what `file_field` and `send_as_attachment` are would be good.